src/app/builders/components/Table.stories.tsx

```diff
@@ -0,0 +1,219 @@
+import type { Meta, StoryObj } from '@storybook/react'
+import { Table } from './Table'
+
+const meta = {
+  title: 'Koto/Backing/Components/Table',
```

Comment:
I would review the stories from the story book to update the title accordingly.
```suggestion
  title: 'Koto/Builders/Components/Table',
```

This is from builders but in the story book is shown under Backing.
![Uploading Screenshot 2025-06-02 at 11.53.03 AM.png…]()